test/MTGCore.Tests/SearchFilterMapperTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using MTGCore.Services;
+using MTGCore.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Shouldly;
+
+namespace MTGCore.Tests
+{
+    public class SearchFilterMapperTests
+    {
+        private SearchFilterMapper mapper = new SearchFilterMapper();
+        private SearchFilterWithColours searchFilter = new SearchFilterWithColours()
+        {
+            Name = "Bomb",
+            Type = "Artifact",
+            Rarity = "Common",
+            Price = "100",
+            Set = "TEST",
+            Colours = new List<string>() {"B","C","U"}
+        };
+
+
+        [Fact]
+        public void returns_list_of_query_string_Parameters()
+        {
+
+            Dictionary<string,string> queryStringList = mapper.map(searchFilter);
+
+
+            var name = queryStringList.Where(x => x.Key == "name").Single();
+            name.Value.ShouldBe("Bomb");
```

Comment:
It's usually quite nice to have the expected values as `const` vars at the top of the test so it's clear what we're expecting the result to be.